### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GauravSarkar**
Signed-off-by: Gaurav Sarkar <60042820+GauravSarkar@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 This repository contains a boilerplate static site generator setup for creating CNCF documentation projects. We strongly recommend using this setup (it helps us help you and your project!), but none of the technologies in the stack are strictly required.
 
 The starter uses the following:
-* **[Hugo](https://gohugo.io/)** as a static site generator
+* **[Hugo(This requires Hugo 9.73+ extended version)](https://gohugo.io/)** as a static site generator
```
Comment (nate-double-u) [inline README.md:6]: Some notes:
* Could you add a space between `Hugo` and the `(`.
* I'm also not sure we need the "This requires Hugo" part, just the version and extended modifier should be enough.
* I also think the original issue had a typo in the version requirement. I think probably we need Hugo's extended v0.73 or above, not version 9.73.
```suggestion
* **[Hugo (extended, v0.73 or above)](https://gohugo.io/)** as a static site generator
```

**Comment by nate-double-u** [comment]
Thanks for looking into this @GauravSarkar, and I'm glad DCO is working for you now.
I've made a couple comments on your proposed changes, and would like to make sure we're referencing the correct version before merging this in.

**Comment by GauravSarkar** [comment]
I have made the required changes. please see.

**Review by celestehorgan — APPROVED** [review]

**Review by nate-double-u — APPROVED** [review]

**Comment by nate-double-u** [comment]
Thanks @GauravSarkar!

**Pull request merged**